### PR TITLE
Update run schema

### LINKED ISSUE #617: api: update schema for task Run
Runs should include task IDs.

We probably don't need `queuedAt`, as the scheduling implementation doesn't queue and pull; it executes as soon as a run-slot is free and the schedule is due to run. Maybe "lag" would be more appropriate (e.g. it was scheduled to run at 1 PM exactly but started running 100ms after). But it's not clear what lag would mean on retries or re-runs of historical time ranges.

I'm not sure what `startTime` and `stopTime` are, as they don't have a description in the swagger. I don't think `startTime` is the `now` value of a run, but we should report that.

### PR DESCRIPTION
**Pull request by jademcgough**
Closes #617

### CODE REVIEW

File: http/swagger.yml [hunk 1]
```diff
@@ -1236,33 +1236,44 @@ components:
...
           enum: [
-            "queued",
+            "scheduled",
             "executing",
             "failed",
             "success"
           ]
-        queuedAt:
+        scheduledFor:
+          description: Time run is scheduled to begin, RFC3339.
```
Comment (mark-rushakoff) [inline http/swagger.yml]: This description isn't quite right. "scheduled to begin" sounds like there's a correlation between wall time and this option, which isn't necessarily the case if the `delay` option is present or if it's a re-run. Maybe `Time used for run's "now" option` would be more accurate?

File: http/swagger.yml [hunk 2]
```diff
@@ -1236,33 +1236,44 @@ components:
...
           format: date-time
         error:
           $ref: "#/components/schemas/Error"
         log:
           readOnly: true
-          description: A url to a relevant log.
+          description: Link to the full logs for a run.
           type: string
-      required: [queuedAt, status]
+      required: [scheduledFor]
```
Comment (mark-rushakoff) [inline http/swagger.yml]: The user doesn't create runs directly, and I don't know how the `required` section affects the schema in swagger, so I'm not sure whether a `required` section makes sense here.

Comment (jademcgough) [reply http/swagger.yml]: Agreed, I'm fine with removing this.

File: http/swagger.yml (hunk 1 above)
Comment (jademcgough) [reply http/swagger.yml]: 👍

**Comment by jademcgough** [comment]
I'll get CI green before this is merged.

**Pull request merged**